.github/pull_request_template.md

```diff
@@ -1,9 +1,9 @@
 Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-
 
-- [ ] Is [CHANGELOG.md](./CHANGELOG.md) extended?
+- [ ] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
 - [ ] Does this PR avoid breaking the API? 
     Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
-- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](./developer-documentation)?
+- [ ] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](https://github.com/xenit-eu/alfred-api/blob/master/developer-documentation)?
 - [ ] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
```

Comment:
We should probably publicize the code style guide somewhere external people can view it too.

Reply:
Yes, if only out of principle (no-one will ever commit on apix), but out of scope for this ticket.